JungleBus/Aws/Sqs/SqsQueue.cs

```diff
@@ -87,7 +85,7 @@ public SqsQueue(RegionEndpoint endpoint, string queueName, int retryCount, IMess
                 _simpleQueueService.SetQueueAttributes(createResponse.QueueUrl, new Dictionary<string, string>() { { "MessageRetentionPeriod", "1209600" } });
             }
 
-            MessageParser = messageParser;
+            MessageParser = new MessageParser();
             MaxNumberOfMessages = 10;
```

Comment:
Looking at this, pretty sure this is a bug and should be hard coded to 1. If the SQS returns more than 1 message, all messages would have the same timeout span. I think this is from when I have multithreading implemented via a single message pump. I will put in a bug for this.